### PR TITLE
Network breadcrumbs README files

### PR DESCRIPTION
**Pull request by lemnik**
## Goal
Include `README.md` files for the new network breadcrumbs packages. These packages will appear independently from our main `bugsnag_flutter` package on pub.dev, and should explain what they're used for in their README files.

### CODE REVIEW

File: packages/bugsnag_breadcrumbs_dart_io/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Network Breadcrumbs for Bugsnag Flutter
+
+HTTP networking breadcrumbs for [bugsnag-flutter](https://pub.dev/packages/bugsnag_flutter) when using the `dart:io` [HttpClient](https://api.flutter.dev/flutter/dart-io/HttpClient-class.html).
```
Comment (tomlongridge) [inline packages/bugsnag_breadcrumbs_dart_io/README.md:3]:
```suggestion
HTTP networking breadcrumbs for [bugsnag_flutter](https://pub.dev/packages/bugsnag_flutter) when using the `dart:io` [HttpClient](https://api.flutter.dev/flutter/dart-io/HttpClient-class.html).
```

File: packages/bugsnag_breadcrumbs_dart_io/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+If you are using the [http](https://pub.dev/packages/http) package then [bugsnag_breadcrumbs_http](https://pub.dev/packages/bugsnag_breadcrumbs_http) should be used for networking breadcrumbs.
+
+## Getting Started
+
+1. Install [bugsnag-flutter](https://pub.dev/packages/bugsnag_flutter)
```
Comment (tomlongridge) [inline packages/bugsnag_breadcrumbs_dart_io/README.md:8]:
```suggestion
1. Install [bugsnag_flutter](https://pub.dev/packages/bugsnag_flutter)
```

File: packages/bugsnag_breadcrumbs_http/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Network Breadcrumbs for Bugsnag Flutter
+
+[http](https://pub.dev/packages/http) networking breadcrumbs for [bugsnag-flutter](https://pub.dev/packages/bugsnag_flutter).
```
Comment (tomlongridge) [inline packages/bugsnag_breadcrumbs_http/README.md:3]:
```suggestion
[http](https://pub.dev/packages/http) networking breadcrumbs for [bugsnag_flutter](https://pub.dev/packages/bugsnag_flutter).
```

File: packages/bugsnag_breadcrumbs_http/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+If you are using the [HttpClient](https://api.flutter.dev/flutter/dart-io/HttpClient-class.html) class in `dart:io` then [bugsnag_breadcrumbs_dart_io](https://pub.dev/packages/bugsnag_breadcrumbs_dart_io) should be used for networking breadcrumbs.
+
+## Getting Started
+
+1. Install [bugsnag-flutter](https://pub.dev/packages/bugsnag_flutter)
```
Comment (tomlongridge) [inline packages/bugsnag_breadcrumbs_http/README.md:8]:
```suggestion
1. Install [bugsnag_flutter](https://pub.dev/packages/bugsnag_flutter)
```

File: packages/bugsnag_breadcrumbs_dart_io/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Network Breadcrumbs for Bugsnag Flutter
+
+HTTP networking breadcrumbs for [bugsnag-flutter](https://pub.dev/packages/bugsnag_flutter) when using the `dart:io` [HttpClient](https://api.flutter.dev/flutter/dart-io/HttpClient-class.html).
+If you are using the [http](https://pub.dev/packages/http) package then [bugsnag_breadcrumbs_http](https://pub.dev/packages/bugsnag_breadcrumbs_http) should be used for networking breadcrumbs.
```
Comment (tomlongridge) [inline packages/bugsnag_breadcrumbs_dart_io/README.md:4]: I think you've got the dart:io and http content switched around here.

Comment (lemnik) [reply packages/bugsnag_breadcrumbs_dart_io/README.md:4]: The second line of each of these ("If you are using...") is a reference to the other package, to keep it clear that there are two ways we support this. If someone opens the `dart:io` package, I think we want them to know we also have support for `http`?

Comment (tomlongridge) [reply packages/bugsnag_breadcrumbs_dart_io/README.md:4]: 🤦 Sorry - yes, I see: I didn't review it in context properly just noticed the reference to the other one at the end!

**Pull request merged**